corehq/toggles.py

```diff
@@ -739,6 +739,13 @@ def _commtrackify(domain_name, toggle_is_enabled):
     save_fn=_commtrackify,
 )
 
+NON_COMMTRACK_LEDGERS = StaticToggle(
+    'non_commtrack_ledgers',
+    "Enable ledgers for projects which don't use Supply.",
```

Comment:
Grammar police here: should say "...projects _that_ don't use Supply." or maybe even "...projects _that do not_ use Supply."